src/main/java/bio/terra/workspace/service/resource/controlled/gcp/ControlledGcsBucketResource.java

```diff
@@ -104,4 +104,13 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash(super.hashCode());
   }
+
+  public static GoogleBucketStoredAttributes attributesToOutputApiModel(String jsonAttributes) {
+    try {
+      return OBJECT_MAPPER.readValue(jsonAttributes, GoogleBucketStoredAttributes.class);
```

Comment:
If I understand correctly, this method is for converting a json-formatted string field in the dao DB to a Java ApiModel object.
- Please add a method comment.
- It feels like this conversion should be in the dao class. Since the conversion from DB  row(s) to Java class (ControlledResourceDbModel) happens there -- is there a reason to pull out the conversion of this particular field?

Reply:
I put it in this class to be a bookend to the `getJsonAttributes()` instance method it has. The knowledge of this part of the conversion in that direction lives in this class already. 

There will be an opportunity to revisit this with the new schema changes.